index.js

```diff
@@ -1,173 +1,31 @@
-    
-    this.reminders = new Enmap({ provider: new EnmapLevel({ name: "reminders" }) });
-
-    this.points = new Enmap({ provider: new EnmapLevel({ name: "points" }) });
-    this.store = new Enmap({ provider: new EnmapLevel({ name: "shop" }) });
-    this.inventory = new Enmap({ provider: new EnmapLevel({ name: "inventory" }) });
-  }
-
-  permlevel(message) {
-    let permlvl = 0;
-
-    const permOrder = client.config.permLevels.slice(0).sort((p, c) => p.level < c.level ? 1 : -1);
-
-    while (permOrder.length) {
-      const currentLevel = permOrder.shift();
-      if (message.guild && currentLevel.guildOnly) continue;
-      if (currentLevel.check(message)) {
-        permlvl = currentLevel.level;
-        break;
-      }
-    }
-    return permlvl;
-  }
-
-  loadCommand(commandPath, commandName) {
-    try {
-      const props = new (require(`${commandPath}${path.sep}${commandName}`))(client);
-      props.conf.location = commandPath;
-      if (props.init) {
-        props.init(client);
-      }
-      client.commands.set(props.help.name, props);
-      props.conf.aliases.forEach(alias => {
-        client.aliases.set(alias, props.help.name);
-      });
-      return false;
-    } catch (e) {
-      return `Unable to load command ${commandName}: ${e}`;
-    }
-  }
-
-  async unloadCommand(commandPath, commandName) {
-    let command;
-    if (client.commands.has(commandName)) {
-      command = client.commands.get(commandName);
-    } else if (client.aliases.has(commandName)) {
-      command = client.commands.get(client.aliases.get(commandName));
-    }
-    if (!command) return `The command \`${commandName}\` doesn"t seem to exist, nor is it an alias. Try again!`;
-
-    delete require.cache[require.resolve(`${commandPath}${path.sep}${commandName}.js`)];
-    return false;
-  }
-
-  getSettings(id) {
-    const defaults = client.settings.get("default");
-    let guild = client.settings.get(id);
-    if (typeof guild != "object") guild = {};
-    const returnObject = {};
-    Object.keys(defaults).forEach((key) => {
-      returnObject[key] = guild[key] ? guild[key] : defaults[key];
-    });
-    return returnObject;
-  }
-
-  writeSettings(id, newSettings) {
-    const defaults = client.settings.get("default");
-    let settings = client.settings.get(id);
-    if (typeof settings != "object") settings = {};
-    for (const key in newSettings) {
-      if (defaults[key] !== newSettings[key]) {
-        settings[key] = newSettings[key];
-      } else {
-        delete settings[key];
-      }
-    }
-    client.settings.set(id, settings);
-  }
-}
+require(`${process.cwd()}/extenders/Message.js`);
+require(`${process.cwd()}/extenders/GuildMember.js`);
+require(`${process.cwd()}/extenders/Guild.js`);
+require(`${process.cwd()}/extenders/DMChannel.js`);
+require(`${process.cwd()}/extenders/TextChannel.js`);
+const MisakiClient = require("./structures/MisakiClient.js");
```

Comment:
This needs to conform to current practices.

Reply:
Im not sure why you use `${process.cwd()}` in require but its so not needed + breaks vs code intellisense

Reply:
When PR'ing in to an existing project, you should conform to what (minimal) standards they have, and this is one of the standards the project uses, pretty much every file uses `process.cwd()`, please change to conform.

Reply:
As agreed to on discord, I'll change the current style of using `${process.cwd}` back to the standard `./` and `../`